MaxText/inference_microbenchmark.py

```diff
@@ -106,7 +109,7 @@ def prefill_insert_benchmark(
       f"\tPrefill + Insert step average time: {prefill_insert_average_ms:.3f} ms\n\n\n\n"
   )
   result_dict = {
-      "prefill_insert_time_in_ms": prefill_insert_average_ms
+      "insert_time_in_ms": prefill_insert_average_ms
```

Comment:
This one is a special case, because I can not get it to time the insert only without prefill.

So this particular metrics is the combination of prefill and insert, i.e. prefill-insert

Reply:
Done.